dev-preview/README.md

```diff
@@ -387,3 +388,13 @@ Currently, there are two variants of Right Sizing available in dev-preview:
 - **Virtualization/VM Level**: Focuses on resource insights at the VM level.
 
 To understand how this can help you optimize resources across multiple clusters, as well as learn about installation steps, detailed usage, and how to start using the feature, you can refer to the full documentation [here](https://github.com/stolostron/right-sizing/blob/main/README.md).
+
+## Image-Based Break/Fix (IBBF) for SNO Hardware Replacement
+
+The Image-Based Break/Fix (IBBF) feature streamlines Single Node OpenShift (SNO) hardware replacement, minimizing downtime while preserving the cluster’s original identity. It ensures that critical details, including cluster identifiers, cryptographic keys (such as kubeconfig), and authentication credentials, are retained. This allows the replacement node to seamlessly assume the identity of the failed hardware.
+
+Designed for like-for-like hardware replacements and IBI-installed SNOs, IBBF introduces a GitOps-compatible, declarative API, enabling users to initiate hardware replacement with a single Git commit. Powered by the SiteConfig Operator and IBI Operator, this functionality allows clusters to be redeployed using the existing ClusterInstance CR. The SiteConfig Operator provides a flexible reinstallation service that integrates seamlessly with template-driven provisioning. Additionally, it includes a backup and restore mechanism for Secrets and ConfigMaps, ensuring the cluster retains its identity.
```

Comment:
I don't remember if it's the SiteConfig Operator that has the backup and restore mechanism but I'd clarify what is "it".
```suggestion
Designed for like-for-like hardware replacements and IBI-installed SNOs, IBBF introduces a GitOps-compatible, declarative API that enables users to initiate hardware replacement with a single Git commit. Powered by the SiteConfig Operator and IBI Operator, this functionality allows clusters to be redeployed by using the existing ClusterInstance CR. The SiteConfig Operator provides a flexible reinstallation service that integrates seamlessly with template-driven provisioning. Additionally, the SiteConfig Operator includes a backup and restore mechanism for Secrets and ConfigMaps, ensuring the cluster retains its identity.
```